Source/FullScreenPokemon.d.ts

```diff
@@ -845,6 +845,12 @@ declare module FullScreenPokemon {
          * Wild Pokemon that may appear in this Area.
          */
         wildPokemon: IAreaWildPokemonOptionGroups;
+
+        /**
+         * Whether the Player has encountered a Pokemon in this area's grass
+         * while the Nuzlocke Challenge mod has been enabled.
```

Comment:
Don't mention the Nuzlocke Challenge mod. They shouldn't be part of the core functionality.